test/utils/fieldUtils.js

```diff
@@ -93,24 +93,24 @@ class FieldDataTypes {
   }
 
   /**
-   * Select a random option from the dropdown
+   * Select a provided option from the dropdown
    * @returns CCDStringField Object
    */
-  async selectFromFixedList(){
+  async selectFromFixedList(value, id){
     let css = await FIELDS.FIXED_LIST.cssTag;
-    let fixedListField = await new CCDFixedListField(css);
-    await fixedListField.selectOption();
+    let fixedListField = await new CCDFixedListField(css, id);
+    await fixedListField.selectOption(value);
     return fixedListField;
   }
 
   /**
-   * Select random radio butto option
+   * Select a provided radio butto option
```

Comment:
or random

Reply:
perhaps we should explicitly state in the doc here that if the value parameter is left blank then a random option will be chosen